### ISSUE TITLE
Library Usage

### ISSUE THREAD

**Issue by SriKrishnaCovalense** [karabaralex/android-comboseekbar#17]
Hi karabralex, please provide the sample code implementation for the three Samples shown in the Illustration. The Sample Module only contains one usage and that too a very simplistic one ! We are unable to use your library because of that.

**Comment by markm1992** [comment]
Agreed. Been trying to figure this out for a while. I've gotten a rudimentary bar working but the rendering is terrible and it never shows the text and the thumb is about 2 thumbs to the right of where it should be

**Comment by SriKrishnaCovalense** [comment]
Thanks Mark. I will too look into the code and will experiment it, will report any issues with the code and UI.

**Comment by karabaralex** [comment]
Ok, Sri. This is really old project, but I will try to recreate this samples
